Makefile

```diff
@@ -27,6 +27,7 @@ KAR_JS_SDK=$(DOCKER_IMAGE_PREFIX)kar-sdk-nodejs-v12:$(DOCKER_IMAGE_TAG)
 KAR_JAVA_SDK=$(DOCKER_IMAGE_PREFIX)kar-sdk-java-builder-11:$(DOCKER_IMAGE_TAG)
 KAR_JAVA_RUNTIME=$(DOCKER_IMAGE_PREFIX)kar-sdk-java-runtime-11:$(DOCKER_IMAGE_TAG)
 KAR_JAVA_REACTIVE_RUNTIME=$(DOCKER_IMAGE_PREFIX)kar-sdk-java-reactive-runtime-11:$(DOCKER_IMAGE_TAG)
+KAR_PYTHON_SDK=$(DOCKER_IMAGE_PREFIX)kar-sdk-python-v1:$(DOCKER_IMAGE_TAG)
```

Comment:
really nitpicking now 😉  why `kar-sdk-python-v1` instead of `kar-sdk-python` or `kar-sdk-python-3.8`?

Reply:
No worries! :) Fixed!